build.gradle.kts

```diff
@@ -89,45 +53,13 @@ tasks {
             kotlinOptions.jvmTarget = "1.8"
         }
     }
-
-    withType<Detekt> {
-        jvmTarget = "1.8"
-    }
-
     patchPluginXml {
-        version(pluginVersion)
-        sinceBuild(pluginSinceBuild)
-        untilBuild(pluginUntilBuild)
-
-        // Extract the <!-- Plugin description --> section from README.md and provide for the plugin's manifest
-        pluginDescription(
-            closure {
-                File("./README.md").readText().lines().run {
-                    val start = "<!-- Plugin description -->"
-                    val end = "<!-- Plugin description end -->"
-
-                    if (!containsAll(listOf(start, end))) {
-                        throw GradleException("Plugin description section not found in README.md:\n$start ... $end")
-                    }
-                    subList(indexOf(start) + 1, indexOf(end))
-                }.joinToString("\n").run { markdownToHTML(this) }
-            }
-        )
-
-        // Get the latest available change notes from the changelog file
-        changeNotes(
-            closure {
-                changelog.getLatest().toHTML()
-            }
-        )
+        version(version)
+        sinceBuild("193")
+        untilBuild("203.*")
```

Comment:
You may recognize this property. It previously was configured in gradle.properties; I decided to move all information from gradle.properties into the gradle build definition because I think it makes things a lot more understandable for newbies.

Reply:
is this was make the plugin unavailable when CLion became 203.*, from 202.* ?

if so , I wonder if this is required, because the old plugin is always available, even if it should not be anymore :-)

Reply:
I didn't look into that, this is still not resolved.

Reply:
as long as we do not forget, .. it not a problem. 
and even if we forget, we might notice not being available in the store anymore :-)

Reply:
#27